tools/checkpoint_conversion/convert_gemma_checkpoints.py

```diff
@@ -86,7 +86,24 @@ def download_flax_model(handle):
     return kagglehub.model_download(handle)
 
 
-def convert_model(flax_config, vocab_size):
+def convert_model(flax_config, flax_params, vocab_size):
+    kwargs = {}
+    # Hack to infer Gemma 2 config options until Flax actually adds support.
+    if "post_attention_norm" in flax_params["transformer"]["layer_0"]:
+        # The 27B parameter model is the only model that does a weird
+        # query normalization.
+        query_head_dim_normalize = flax_config.num_heads != 32
```

Comment:
1. Is the 27b's weird query normalization `hidden_dim/num_query_heads`?
2. I think it'd be easier to read if we have a separate variable for keeping whether it's a 27b model. Something like this:
```
is_gemma2_27b = flax_config.num_heads == 32
query_head_dim_normalize =  !is_gemma2_27b
```

Reply:
sg!